src/osd/PGBackend.cc

```diff
@@ -513,76 +552,81 @@ map<pg_shard_t, ScrubMap *>::const_iterator
     if (i == j->second->objects.end()) {
       continue;
     }
-    if (i->second.read_error || i->second.stat_error) {
-      // scrub encountered read error or stat_error, probably corrupt
-      dout(10) << __func__ << ": rejecting osd " << j->first
-	       << " for obj " << obj
-	       << "," << (i->second.read_error ? " read_error" : "")
-	       << (i->second.stat_error ? " stat_error" : "")
-	       << dendl;
-      continue;
+    string error_string;
+    shard_map[j->first];
```

Comment:
@tchaikov I was going to leave this alone, but I'm happy to add shard_info.  I'm pretty sure I need the shard_map to include the new entry even if no errors are detected, so I can't just remove the line.